UnitsNet/Scripts/UnitDefinitions/ForcePerLength.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "Name": "ForcePerLength",
+  "BaseUnit": "NewtonPerMeter",
+  "XmlDoc": "Force per length is the amount of change applied per length.",
```

Comment:
Agreed. The current wording looks like it might describe the derivative of a distributed load wrt distance, but the units of that would be the same as pressure I believe.